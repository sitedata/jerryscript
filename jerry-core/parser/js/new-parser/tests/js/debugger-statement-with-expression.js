/* This test contains debugger statement parser negative test. */

debugger 0
