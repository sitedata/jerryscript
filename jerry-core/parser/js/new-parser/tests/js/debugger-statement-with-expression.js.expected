
--- Script parsing start ---

  [  0] CBC_EXT_DEBUGGER

--- Script parsing end ---

Parse error 'Expected ';' token.' at line: 3 col: 10
